tests/playwright/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "moduleResolution": "Node",
     "declaration": true,
     "outDir": "dist",
-    "rootDir": "src",
+    "rootDirs": ["src", "../../playwright.config.ts"],
```

Comment:
shouldn't playwright config file be part of this package ?

Reply:
It's not really required, because we need a different playwright config in the extension solution anyway so when I created it initially I put it in the same place the vitest config file was located in order not to change too many things at once.

Reply:
yes but now I think it's time to move it ( we have one in argos folder)
And it does not make sense to have something at the root folder to reference files only in this tests/playwright folder

Reply:
@benoitf I just tried moving the config locally into `tests/playwright` and it becomes very complex very quickly. The issue is that in order for the config to be used it needs to be in the same folder from where the playwright test execution command is issued, which means that if the config is in `tests/playwright` we will also have to move all the relevant scripts from the root `package.json` file, but the build commands for the entire solution are still in the root file, so we'd have to have half of the commands in the root `package.json` and half in the `package.json` in `tests/playwright` even all that I was still not able to get it to run locally right now, since the main entry point into the application is not the same.

While I'm sure there is some configuration that makes it work like this, it's a lot more convoluted than is in scope for this PR.

Reply:
Can't the root package.json commands just delegate to this package.json ?

Reply:
@jeffmaury I'm not sure what you mean by delegate in this context, we can certainly run commands from the root `package.json` towards the `tests/playwright` `package.json` using the `npm run --prefix path/to/folder <command>` command, however the `<command>` would still have to be located in the `tests/playwright` `package.json` so it would still involve splitting all the scripts.

But in addition to that the `main` entry point into the app is not the same, when I use the one that is currently present in the `tests/playwright` the PD app doesn't even start up and if I replace and adapt it to the one that is in the root `package.json` I get errors like these:

![image](https://github.com/user-attachments/assets/38c8fd23-dab4-467c-a1ba-89f4ed281967)